libbeat/dashboards/import_dashboards.go

```diff
@@ -677,11 +676,13 @@ func main() {
 	if importer.cl.opt.Dir != "" {
 		if err = importer.ImportKibana(importer.cl.opt.Dir); err != nil {
 			fmt.Println(err)
+			os.Exit(1)
```

Comment:
I agree that errors should be going to stderr. That said, I think this PR is an improvement as-is, since it at least standardizes the exit codes on error.